common/src/main/scala/com/aliyun/emr/rss/common/RssConf.scala

```diff
@@ -923,6 +923,14 @@ object RssConf extends Logging {
     conf.getTimeAsMs("rss.worker.partitionSorterCloseAwaitTime", "120s")
   }
 
+  def waitCleanTaskSubmitBeforeCloseTimeoutMs(conf: RssConf): Long = {
+    conf.getTimeAsMs("rss.worker.waitCleanTaskSubmitTimeoutMs", "500ms")
+  }
+
+  def workerDiskFlusherShutdownTimeoutMs(conf: RssConf): Long = {
+    conf.getTimeAsMs("rss.worker.diskFlusherShutdownTimeoutMs", "3000ms")
```

Comment:
Seems we should also increase `rss.worker.shutdown.timeout`

Reply:
i think current timeout which is 600s may be enough, in previous test this delete process could be done in 1s for 4~500GB file for one diskOperator on SSD.